modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -11,7 +11,62 @@ Before you install an {product-title} cluster, review the following Nutanix AOS
 [id="installation-nutanix-installer-infra-reqs-account_{context}"]
 == Required account privileges
 
-Installing a cluster to Nutanix requires an account with administrative privileges to read and create the required resources.
+The installation program requires access to a Nutanix account with the necessary permissions to deploy the cluster and to maintain the daily operation of it. The following options are available to you:
+
+* You can use a local Prism Central user account with administrative privileges. Using a local account is the simplest way to grant access to an account with the required permissions.
```

Comment:
Should "simplest" be stripped out for something like "quickest way"? According to the IBM SG:

![Screenshot from 2023-08-03 09-58-31](https://github.com/openshift/openshift-docs/assets/57954076/59f65e1a-7a08-437c-b0ae-0444da6ca411)

See, [Claims and recommendations](https://www.ibm.com/docs/en/ibm-style?topic=information-claims-recommendations)

Reply:
Good point. Thank you.